src/go/cmd/api_view.go

```diff
@@ -149,6 +150,14 @@ func loadPackage(dir string) (pkg pkg, err error) {
 		err = fmt.Errorf("didn't find any packages in '%s'. Length: %d", dir, len(packages))
 		return
 	}
+	// remove test packages
+	if len(packages) == 2 {
```

Comment:
Instead of deleting them, in your call to `parser.ParseDir()` you can set a filter to skip test files.  See [here](https://github.com/Azure/azure-sdk-for-go/blob/master/tools/profileBuilder/model/transforms.go#L78) for how we do it in the profile builder.

Reply:
Done